bin/run-nextclade

```diff
@@ -11,11 +11,17 @@ trap "exit" INT
 : "${2:?\"[ERROR] ${0}: Output TSV filename as the 2nd argument is required.\"}"
 : "${3:?\"[ERROR] ${0}: Directory for Nextclade input data as the 3rd argument is required.\"}"
 : "${4:?\"[ERROR] ${0}: Output directory as the 4th argument is required.\"}"
+: "${5:?\"[ERROR] ${0}: Output fasta filename as the 5th argument is required.\"}"
+: "${6:?\"[ERROR] ${0}: Output insertions csv filename as the 6th argument is required.\"}"
+: "${7:?\"[ERROR] ${0}: List of genes as 7th argument is required.\"}"
 
 INPUT_FASTA="${1}"
 OUTPUT_TSV="${2}"
 NEXTCLADE_DATASET_DIR="${3}" # external data required for nextclade to function will be downloaded there
 NEXTCLADE_OUTPUTS_DIR="${4}" # files other than TSV will be written by Nextclade there (aligned sequences, peptides, etc.)
+OUTPUT_FASTA="${5}"
+OUTPUT_INSERTIONS="${6}"
+GENES="${7}"
```

Comment:
`run-nextclade` script have gained 3 parameters:
 - OUTPUT_FASTA - "data/genbank/nextclade.aligned.new.fasta" - this is where the newly aligned sequences end up
 - OUTPUT_INSERTIONS - "data/genbank/nextclade.insertions.csv" - same for insertions (needed for `mutation-summary` script). The filename does not mention that they are new, because there is no "old" corresponding file, i.e. we don't store these on S3.
 - GENES - list of genes - "${GENES}" variable in the caller scripts -  This can be moved into config, similarly to how ncov does it. Note that Nexclade is using comma-delimited list, while `mutation-summary` script prefers space-delimited for some reason. We might make `mutation-summary` to reconsider its poor and random behavior :)  

The entire `bin/run-nextclade` should go away soon, because all it does is downloading the dataset and runs nextclade with the same exact parameters. It only confuses things.